lib/ctcs2_to_junit.pm

```diff
@@ -93,11 +93,11 @@ sub generateXML {
 
         $writer->startTag(
             'testcase',
-            classname => get_var("QA_TESTSUITE"),
+            classname => get_var('QA_TESTSET') || get_var('XFSTESTS'),
```

Comment:
if `QA_TESTSET` or `XFSTESTS` is not set, classname will be `false`, but I assume any of those 2 variables are mandatory

Reply:
Indeed, these two parameters are mandatory in different tests could call this function.